packages/react-core/src/components/SearchInput/SearchInput.tsx

```diff
@@ -0,0 +1,92 @@
+import * as React from 'react';
+import styles from '@patternfly/react-styles/css/components/SearchInput/search-input';
+import { css } from '@patternfly/react-styles';
+import { Button, ButtonVariant } from '../Button';
+import { Badge } from '../Badge';
+import AngleDownIcon from '@patternfly/react-icons/dist/js/icons/angle-down-icon';
+import AngleUpIcon from '@patternfly/react-icons/dist/js/icons/angle-up-icon';
+import TimesIcon from '@patternfly/react-icons/dist/js/icons/times-icon';
+import SearchIcon from '@patternfly/react-icons/dist/js/icons/search-icon';
+
+export interface SearchInputProps extends Omit<React.HTMLProps<HTMLDivElement>, 'onChange' | 'results'> {
+  /** Additional classes added to the banner */
+  className?: string;
+  /** Value of the search input */
+  value?: string;
+  /** The number of search results returned. Either a total number of results,
+   * or a string representing the current result over the total number of results. i.e. "1 / 5" */
+  numberOfResults?: number | string;
```

Comment:
The `numberOfResults` name implies number is the type. Perhaps this be something more like "resultsCount", "resultsTotal", or "resultsText"?